src/Process.php

```diff
@@ -45,10 +45,15 @@ class Process extends EventEmitter
     * @param string $cwd     Current working directory or null to inherit
     * @param array  $env     Environment variables or null to inherit
     * @param array  $options Options for proc_open()
+    * @throws LogicException On windows
     * @throws RuntimeException When proc_open() is not installed
     */
     public function __construct($cmd, $cwd = null, array $env = null, array $options = array())
     {
+        if (substr(strtolower(PHP_OS), 0, 3) === 'win') {
+            throw new \LogicException('Windows isn\'t supported.');
```

Comment:
The exception message could be a little bit more descriptiove, something like "Windows isn't supported due to the blocking nature of STDIN/STDOUT/STDERR pipes."